### ISSUE TITLE
on button click open pick an app, but cypress considers it as opening  new page url

### ISSUE THREAD

**Issue by ghost** [cypress-io/cypress#20385]
### Current behavior

on click of the button it calls below open pick an app in browser
![image](https://user-images.githubusercontent.com/99789295/155930799-77091a56-7e32-4f0a-868f-103be06ab08a.png)

In the application , they used window.location.assign = 'tel:98372638783'.

cypress considering it as new page load.

### Desired behavior

Need to handle these kind of calls, instead of checking for the new page load.

### Test code to reproduce

<html>
    <head>
        <title>URL Scheme Navigation example</title>
    </head>
    <body>
        <script type="text/javascript">

            function clickme() {
               window.location.assign = "tel:98372638783"
            }
        </script>
        <button onclick="clickme()" id="clickme" data-at="clickme" value="Click me!">Click Me!</button>
    </body>
</html>

describe('testing', () => {
    
    it('test the clictocall', { baseUrl: null }, () => {
      cy.visit('./url_scheme.html'); 
      cy.get('#clickme').click();
    });
  });
  

### Cypress Version

9.5.0

**Comment by jbpallingayan** [comment]
up on this. experiencing this now. cy.on('window:prompt/alert') cant handle this.

**Comment by metalocator** [comment]
Similar experience with testing "tel:" links in our application.  Removing target attribute does not help.

**Comment by bvogel** [comment]
Yeah, just bc no developer cared to take a look it got closed and not ever fixed. Now facing this as well...

**Comment by jennifer-shehane** [comment]
Sorry for not seeing this earlier. I thought we had an issue open around these types of App links. They're not really currently supported right now. We'd have to add some logic to support it properly.

**Comment by zinaemilian** [comment]
how can I verify that 'open pick an app' is displaying, at least for tseting?
